internal/synclet/synclet.proto

```diff
@@ -5,20 +5,24 @@ package synclet;
 option go_package = "github.com/windmilleng/tilt/internal/synclet/proto;proto";
 
 service Synclet {
-    rpc GetContainerIdForPod (GetContainerIdForPodRequest) returns (GetContainerIdForPodReply) {}
+    rpc GetContainerIdForPod (GetContainerIdForPodRequest) returns (stream GetContainerIdForPodReply) {}
 
     // updates the specified container and then restarts it
     // (much functionality packed into one rpc to minimize latency)
-    rpc UpdateContainer (UpdateContainerRequest) returns (UpdateContainerReply) {}
+    rpc UpdateContainer (UpdateContainerRequest) returns (stream UpdateContainerReply) {}
 }
 
 message GetContainerIdForPodRequest {
     string pod_id = 1;
```

Comment:
not _really_ related to this pr, but is there a reason you split this up into two RPCs, rather than putting the pod_id in the UpdateContainer RPC?

Reply:
I don't recall a reason, and looking at the code now, I don't see a reason we need the container ID in the client. Maybe the code has changed, or maybe we were just paralleling the way the local container builds worked? AFAICT, the only thing we use the container ID for is to pass it back into UpdateContainer. I don't want to block this PR on that. I'll double-check with @maiamcc when she's back.

Reply:
ah, this lets us start paying the cost of finding the containerID in the background as soon as the first build finishes, rather than synchronously before the second build starts